Sources/Playbook/SnapshotSupport/SnapshotSupport.swift

```diff
@@ -71,73 +71,132 @@ public enum SnapshotSupport {
             handler(resource.renderer.image(actions: resource.actions))
         }
     }
-}
-
-private extension SnapshotSupport {
-    struct Resource {
-        var renderer: UIGraphicsImageRenderer
-        var actions: UIGraphicsDrawingActions
-    }
-
-    static func makeResource(
+    
+    /// Generates an `UIView` that snapshots the given scenario.
+    ///
+    /// - Parameters:
+    ///   - scenario: A scenario to be snapshot.
+    ///   - device: A snapshot environment simulating device.
+    ///   - keyWindow: The key window of the application.
+    ///   - handler: A closure that to handle generated `UIView`.
+    ///
+    /// - Note: Passing the key window adds the scenario content to the view
+    ///         hierarchy and actually renders it, so producing a more accurate
+    ///         snapshot image.
+    public static func view(
         for scenario: Scenario,
         on device: SnapshotDevice,
-        scale: CGFloat,
-        keyWindow: UIWindow?,
-        completion: @escaping (Resource) -> Void
+        keyWindow: UIWindow? = nil,
+        handler: @escaping (UIView) -> Void
     ) {
         withoutAnimation {
             let window = SnapshotWindow(scenario: scenario, device: device)
             let contentView = window.contentView!
 
-            let isEmbedInKeyWindow: Bool
-
             if let keyWindow = keyWindow {
                 keyWindow.addSubview(window)
-                isEmbedInKeyWindow = true
-            }
-            else {
-                isEmbedInKeyWindow = false
             }
 
             window.prepareForSnapshot {
                 if contentView.bounds.size.width <= 0 {
-                    fatalError("The view did laid out with zero width in scenario - \(scenario.name)", file: scenario.file, line: scenario.line)
+                    fatalError("The view was laid out with zero width in scenario - \(scenario.name)", file: scenario.file, line: scenario.line)
                 }
 
                 if contentView.bounds.size.height <= 0 {
-                    fatalError("The view did laied out with zero height in scenario - \(scenario.name)", file: scenario.file, line: scenario.line)
+                    fatalError("The view was laid out with zero height in scenario - \(scenario.name)", file: scenario.file, line: scenario.line)
                 }
+            
+                handler(window.scenarioView)
```

Comment:
This is changed from (previously used) contentView to scenarioView because of compatibility with AccessibilitySnapshot. This change doesn't seem to affect any generated snapshots in test.

Reply:
So, can we remove the `contentView` and use `scenarioView` instead of that?

Reply:
I've decided to keep this as contentView so that snapshots are not affected. The accessibility plugin will just use the superview of contentView instead.